### DISCUSSION TITLE
Barely see nothing in GitHub

### DISCUSSION THREAD

**Discussion by 0123456789-jpg** [community/community#113442] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

<img width="1300" alt="截屏2024-03-20 22 37 04" src="https://github.com/community/community/assets/56025753/8dfc6e43-e7f6-4dca-87a5-2c04e1757bf4">
Just visited GitHub website and this is what i see.
After forcely changing to the light theme by editing DOM, i found that most styles are broken (borders, backgrounds, etc.).
I'm using a fairly new Safari, so I think this might be a website issue.

**Comment by 0123456789-jpg** [comment]
ps. I'm sending this from another not-daily-used browser cuz' every page on GitHub is affected.

**Comment by ghostinhershell** [comment]
Hi @0123456789-jpg, 

Thanks so much for reaching out in the community. I would like to recreate this on my end, if possible. Are you using a particular theme here or do you have any reproductions steps I can follow?

**Reply by njneeteson** [reply]
I am also having this issue:

![image](https://github.com/community/community/assets/24441644/cddf96c3-0e43-41a9-9d46-d8567e7ba636)

Every github page looks like this in Safari. 

It started happening to me yesterday. Everything was fine and then some pop-up happened in the top-right of the screen to notify me about something. I could not interact with the pop-up and when I reloaded every page looked like this. I did not change my theme or any of my settings, so there aren't really reproduction steps from the user end. It just happened all of a sudden one time when I opened github.

**Comment by njneeteson** [comment]
It seems to affect all dark themes. I used chrome to change my theme to light and I can see stuff again. It would be nice to be able to use a dark theme because this light theme is hard on the eyes but using the light theme at least makes github useable on safari for now

**Reply by njneeteson** [reply]
Update: it affects light themes as well.

Chrome:
![image](https://github.com/community/community/assets/24441644/76b6620a-ff68-4ec2-9712-6f59541213d9)

Safari:
![image](https://github.com/community/community/assets/24441644/d6377e34-c225-4fba-a915-658f7721b46d)

There are a bunch of visual differences between the two, things missing in the Safari browser. e.g. you can't see the little pip that tells you when you have unread notifications. In Safari I also cannot open the menu that pops out when you click your profile image in the top right of the page (can open it with Chrome)

**Reply by 0123456789-jpg** [reply]
Exactly same.

**Comment by ghostinhershell** [comment]
I really appreciate you both following up with information on what you're seeing on your end. I'm still not seeing the same on mine, unfortunately 😞 

Can you both try clearing your browsing history and cookies on Safari, then try viewing the page again, to see if that resolves this issue? 
- [How to Clear Safari Browsing History and Cache](https://www.macrumors.com/how-to/clear-safari-cache/)

**Reply by njneeteson** [reply]
I followed the link and the instructions and deleted all content related to github. After logging in again on Safari, no change. Still broken.

**Comment by njneeteson** [comment]
Here is the JS console while I look at a page and try to click on the thing in the top right to get the user account menu and the search bar at the top (both unresponsive):

![image](https://github.com/community/community/assets/24441644/613cf191-af72-4ee0-a101-1dfbe486524c)

**Comment by njneeteson** [comment]
For me this appears to be solved now by updating my Mac to Sonoma and Safari to version 17.4.

**Comment by 0123456789-jpg** [comment]
I also tried clearing cache, etc. on my Safari, but still don't work. It's weird that my JS console doesn't emit any error.
ps. @njneeteson 's JS console error can be solved by enabling `Dialog Element` in experimental feature menu.

**Comment by cehfisher** [comment]
Hi @0123456789-jpg - I wanted to follow up on this issue. Is this issue resolved for you using Safari? If yes, could you please close the issue? If no, please add any additional instructions to reproduce your issue and include screenshots and/or URLs, if applicable. Thank you! (cc @queenofcorgis)
